src/amo/components/AddonsCard/styles.scss

```diff
@@ -20,6 +20,18 @@
       border: 0;
       position: absolute;
       top: 0;
+
+      @include respond-to(large) {
+          [dir='ltr'] & {
+              padding-left:0;
```

Comment:
We have a mixin called [`padding-start`](https://github.com/mozilla/addons-frontend/blob/master/src/core/css/inc/mixins.scss#L175-L183), which you can use that will take care of dealing with `ltr` vs `rtl` for you.

Reply:
It works perfectly with that mixin, thank you!